apps/dashboard/src/main/java/com/akto/action/ApiCollectionsAction.java

```diff
@@ -110,6 +134,95 @@ public String deleteMultipleCollections() {
         return SUCCESS.toUpperCase();
     }
 
+    public String fetchSensitiveInfoInCollections(){
+        this.sensitiveInfoInApiCollections = SingleTypeInfoDao.instance.getSensitiveInfoForCollections() ;
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String fetchCoverageInfoInCollections(){
```

Comment:
can we combine some of these into single api?

Reply:
The calls in frontend are handled differently and they are fired parallelly.